homeassistant/helpers/device_registry.py

```diff
@@ -95,6 +73,32 @@ def disabled(self) -> bool:
         return self.disabled_by is not None
 
 
+@attr.s(slots=True, frozen=True)
+class DeletedDeviceEntry:
+    """Deleted Device Registry Entry."""
+
+    id: str = attr.ib()
```

Comment:
Why do we change the order of the defined attributes? It's easier to review if we only change things that are required to change.

Reply:
I seem to remember that's a relic from a time/WIP I was marking some of the fields `Optional` and there were failures related to non-optional args defined after optional ones. But seems it's no longer needed, will revert that one back.